.github/actions/copm_test/action.yaml

```diff
@@ -1,8 +1,22 @@
 name: 'Copm Test'
 description: "common setup steps for the weaver network needed to run copm tests"
+inputs:
+  github-actor:  
+    description: the github actor variable
+    required: true
+  github-token:
+    description: the github secret token variable
+    required: true
 runs:
   using: "composite"
   steps:
+    - name: free github runner disk space
+      shell: bash
+      run: |
+          sudo rm -rf /usr/share/dotnet
```

Comment:
@jenniferlianne Suggestion: Consider re-using the logic from ci.sh that does the same kind of space clearing. If it's not flexible enough and/or too heavy / too slow then please make a note in the comments here or open an issue on GH with suggestions (for exmaple we could pull the disk space freeing logic to a separate bash script that you can source and then re-use easily)

Reply:
I took a few runs at this but:
- the space must be cleared before building and deploying weaver, and
- as is, the command in ci.sh does not clear up enough space.  
 
I tried adding additional directories to the remove section, however that could impact other jobs -- and the copm_jobs failed regardless on the second call to ci.sh to run the tests. https://github.com/hyperledger-cacti/cacti/actions/runs/11838070296/job/32986450316